core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.19.0.groovy

```diff
@@ -31,4 +31,40 @@ schemaChange {
   author 'jsanda'
   tags '0.19.x'
   cql "ALTER TABLE data WITH compaction = {'class': 'DateTieredCompactionStrategy'} AND default_time_to_live = 604800"
-}
\ No newline at end of file
+}
+
+schemaChange {
+  version '3.2'
+  author 'snegre'
+  tags '0.19.x'
+  cql "ALTER TABLE data DROP aggregates"
```

Comment:
We can add later anything we need to the schema. Right now these are unused for the original purpose and the PR from @jsanda for aggregates will not use them either.